### PR TITLE
update task table_transformer

### LINKED ISSUE #191: Update task table_transformer
Needs to be fixed for task list, show, show-run, list-runs

### PR DESCRIPTION
**Pull request by jaysterp**
Fixes #191

### CODE REVIEW

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py [hunk 1]
```diff
@@ -184,17 +220,40 @@ def _build_format_group(item):
...
+
 def _get_value(item, *args):
     """Recursively get a nested value from a dict.
     :param dict item: The dict object
     """
+
     try:
         for arg in args:
-            item = item[arg]
+            item = item[arg]
```
Comment (ehotinger) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py]: spacing

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py [hunk 2]
```diff
@@ -184,17 +220,40 @@ def _build_format_group(item):
...
     try:
         for arg in args:
-            item = item[arg]
+            item = item[arg]          
         return str(item) if item else ' '
     except (KeyError, TypeError, IndexError):
         return ' '
 
+def _get_value_arr(item, *args):
+    """Recursively get a nested array from a dict.
```
Comment (ehotinger) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py]: nit: no recursion here

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py (hunk 1 above)
Comment (ankurkhemani) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py]: Fixed.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py (hunk 2 above)
Comment (ankurkhemani) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py]: Fixed.

**Pull request merged**